### DISCUSSION TITLE
Unable to Get BLIP Captioning to Work

### DISCUSSION THREAD

**Discussion by poondoggle** [bmaltais/kohya_ss#303] [Q&A] [answered]
Hi All,

I am new to kohya_ss and set it up without issue. The issue I am running into now is I am unable to perform BLIP captioning on my test images.  I have it configured as follows:

<img width="1420" alt="Screenshot 2023-03-05 at 1 48 10 AM" src="https://user-images.githubusercontent.com/3400051/222953419-fceba15d-8901-4cbe-9926-3f3a5db84e97.png">

This is the error from the command line.

Captioning files in D:/AI/lora_training_images/natalieportman/image/150_natalieportman...
./venv/Scripts/python.exe "finetune/make_captions.py" --batch_size="8" --num_beams="10" --top_p="0.9" --max_length="75" --min_length="22" --beam_search --caption_extension=".txt" "D:/AI/lora_training_images/natalieportman/image/150_natalieportman" --caption_weights="https://storage.googleapis.com/sfr-vision-language-research/BLIP/models/model_large_caption.pth"
'.' is not recognized as an internal or external command,
operable program or batch file.

Any ideas on what I'm doing wrong?

**Comment by sitrec** [comment]
Also getting the '.' is not recognized as an internal or external command,
operable program or batch file. error but while trying to extract LoRA

**Comment by ricotam782** [comment]
I also had this problem and install the Required Dependencies seems solved it. 

Install [Python 3.10](https://www.python.org/ftp/python/3.10.9/python-3.10.9-amd64.exe)
make sure to tick the box to add Python to the 'PATH' environment variable
Install [Git](https://git-scm.com/download/win)
Install [Visual Studio 2015, 2017, 2019, and 2022 redistributable](https://aka.ms/vs/17/release/vc_redist.x64.exe)

**Comment by poondoggle** [comment, marked as answer]
Issue was resolved with an update that was made overnight.

**Reply by sitrec** [reply]
Thanks!
